share/smack/top.py

```diff
@@ -175,6 +175,9 @@ def arguments():
   translate_group.add_argument('--float', action="store_true", default=False,
     help='enable bit-precise floating-point functions')
 
+  translate_group.add_argument('--split-structs', action='store_true', default=False,
```

Comment:
Let's make it optional for now since we haven't tested this flag extensively on SVCOMP benchmarks. If the results look fine, we can remove this flag.